crates/schema-api/src/lib.rs

```diff
@@ -86,6 +86,12 @@ pub mod deployment {
             #[cfg_attr(feature = "serde_schema", schemars(with = "String"))]
             address: Uri,
             protocol_type: ProtocolType,
+            #[cfg_attr(
+                feature = "serde",
+                serde(with = "serde_with::As::<restate_serde_util::VersionSerde>")
```

Comment:
I guess this could now be replaced with `with = "http_serde::Version"`.

Reply:
it could, but we dont import that package here, and it would then be different to below (where there is the option). so i would vote not to